R/Spectra.R

```diff
@@ -321,6 +321,27 @@ NULL
 #'   0-intensity peaks next to non-zero intensity peaks are retained while with
 #'   `all = TRUE` all 0-intensity peaks are removed.
 #'
+#' - `compareSpectra`: compare each spectrum in `x` with each spectrum in `y`
+#'   using the function provided with `FUN` (defaults to [cor()]). If `y` is
+#'   missing, each spectrum in `x` is compared with each other spectrum in `x`.
+#'   `FUN` is supposed to be a function to compare intensities of (matched)
+#'   peaks of the two spectra that are compared. The function needs to take two
+#'   numeric vectors as input and is supposed to return a single numeric as
+#'   result. Additional parameters to the function can be passed with `...`.
+#'   Prior to the comparison of the intensities, `compareSpectra` matches peaks
+#'   based on the similarity of their m/z with arguments `tolerance` and `ppm`
+#'   allowing to define the maximal allowed difference of m/z values for peaks
+#'   to be matched: `tolerance` allows to define a constant maximal accepted
+#'   difference and `ppm` a relative, m/z dependent, difference (i.e.
+#'   differences smaller than `ppm` parts-per-million of the actual peak's m/z
+#'   are allowed; see [closest()] for more information).
+#'   The function matches peaks for a spectrum in `x` with peaks
```

Comment:
This sentence doesn't really make sense here? Is there something missing?

Reply:
Thanks! yes, makes no sense. will remove.